content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst

```diff
@@ -120,113 +113,103 @@ proposed price coming from the finished products C.
 Traceability
 ============
 
-In case the products received from the subcontractor contain tracked
+In case the received products from the subcontractor contain tracked
 components, their serial or lot numbers need to be specified during the
-receipt.
+receipt. 
 
-On the receipt of the subcontracted product, a *Record Components*
-button appears when necessary. Click on it to open a dialog and record
-the serial/lot numbers of the components. If the finished product is
-also tracked, its serial/lot number can be registered here too.
+In that case, on the receipt of the subcontracted product, a *Record Components*
+button appears. Click on it to open a dialog box and record the serial/lot numbers of 
+the components. If the finished product is also tracked, its serial/lot number can
+be registered here too.
 
-.. image:: media/subcontracting_05.png
+.. image:: media/SBC_3.png
     :align: center
     :alt: 
 
 For audit purposes, it is possible to check the lot numbers recorded on
 a receipt by using the icon on the right of the finished products:
 
-.. image:: media/subcontracting_06.png
+.. image:: media/SBC_4.png
     :align: center
     :alt: 
 
-Automate Replenishment of Subcontractors
-========================================
-
-To manage the resupply of your subcontractor, activate
-*Multi-locations* in :menuselection:`Inventory --> Configuration -->
-Configuration`. Then, the inventory at the subcontractor location can
-be tracked.
+Also note that in case flexible consumption has been selected on the subcontracted 
+BOM for a non-tracked product, the record components option will also appear 
+optionally on each move line, if you want to register more or less component consumption 
+at your subcontracting location, when receiving your final product. 
 
-It is possible to resupply subcontractors by sending products from the
-company locations, or by sending products from another supplier to the
-subcontractors.
+.. image:: media/SBC_5.png
+    :align: center
+    :alt: 
 
-Replenishment from the warehouse
---------------------------------
+As you can see, the reception of both of these non-tracked products can either be executed 
+by selecting the 'Set Quantities' Option or via the Move Line 'Hamburgers'. 
```

Comment:
I would maybe say "move line hamburger menus" because I was very confused for a few minutes :)